src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -104,6 +105,15 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
     if (!this.state.body) {
       return;
     }
+    if ('sortOrder' in options) {
+      const serviceScene = sceneGraph.getAncestor(this, ServiceScene);
```

Comment:
This looks like it's grabbing the queryRunner descendents of the serviceScene, I think you'll just want to run `sceneGraph.getData(this)`, which should return the closest queryProvider of the logs panel, and then check for SceneQueryRunner descendents on that if it's not already a sceneQueryRunner

Reply:
e.g. 
```
const $data: SceneDataTransformer | SceneQueryRunner =  sceneGraph.getData(this)
const queryRunner = $data instanceof SceneQueryRunner ? $data : sceneGraph.findDescendents($data)[0]
```

Reply:
Worked like a charm 🪄